cosmos/scheduler/cmd/scheduler.go

```diff
@@ -269,14 +273,14 @@ func (s *Scheduler) setSecret(secret *Secret, patch bool) error {
 	return nil
 }
 
-
 // isCosmosLocation checks if a given BucketTransaction is in a supported
 // Cosmos location type. This will also set the LocationType property of 
 // the BucketTransaction based off the location.LocationType. BucketTransaction
 // is expected to be not nil.
 func (s *Scheduler) isCosmosLocation(bucket *BucketTransaction) (bool) {
 	locationTypes := []string{
 		nfsLocation,
+		awsLocation,
```

Comment:
add `cephLocation` ?

Reply:
Ahh yeah good catch. I was just writing the tests and trying to figure out why it wasn't triggering. 😅